subprojects/core/src/main/java/org/gradle/execution/plan/DefaultNodeValidator.java

```diff
@@ -66,6 +66,9 @@ private WorkValidationContext validateNode(LocalTaskNode node) {
     }
 
     private void logWarnings(List<? extends InternalProblem> problems) {
+        // We are logging the warnings that we encountered, so this should be always the next Gradle version.
```

Comment:
❓ 

I am not sure I understand this comment.

"this should be always the next Gradle version" - what is "this" in this context?

"What should happen when preparing the next major version..." - is this a recommendation? Is this a good place to keep it?

Reply:
Yes, that comment doesn't make a lot of sense. 

I removed the comment and I now use `willBecomeAnErrorInNextMajorGradleVersion` instead of `willBecomeAnErrorInGradle10()`, so we don't need to update it.